install/index.yaml

```diff
@@ -24,4 +47,4 @@ entries:
     urls:
     - wavefront-operator-0.1.1.tgz
```

Comment:
Why does this still point to 0.1.1?

Reply:
Line 25 points to the latest version, this was the earlier version. details of both versions are preserved